screengrab/lib/screengrab/runner.rb

```diff
@@ -342,7 +342,11 @@ def move_pulled_screenshots(locale, pull_dir, device_type_dir_name)
         # specific name, as expected by supply
         #
         # (Moved to: fastlane/metadata/android/en-US/images/phoneScreenshots)
-        dest_dir = File.join(File.dirname(dest_dir), locale, 'images', device_type_dir_name)
+        if dest_dir.include?("#{locale}/images")
+          dest_dir = File.join(File.dirname(dest_dir), device_type_dir_name)
+        else
+          dest_dir = File.join(File.dirname(dest_dir), locale, 'images', device_type_dir_name)
+        end
```

Comment:
I commented this in the issue too but I think the solution here needs to be on line 326 https://github.com/fastlane/fastlane/pull/16063/files#diff-693ac675cd5d301f4c33f1e075c624f4R326

```rb
screenshots_dir_pattern = File.join(pull_dir, '**', "screenshots")
```

needs to be something like

```rb
screenshots_dir_pattern = File.join(pull_dir, '**', locale, **, "screenshots")
```
I think 🤔

`move_pulled_screenshots` now gets run multiple times but its pulling out every locale (currently) because `screenshots_dir_pattern` is not filtering by the newly passed in `locale`

Reply:
I'm sorry but I don't understand how the change you provided above explains why for some the resulting path was `/en-US/images/phoneScreenshots` and for some just `/phoneScreenshots`. The issue was not that images were overridden, the issue was the path itself. At least I don't understand how the suggestion you make would influence the resulting path ...